terraform/aws/modules/aws_instance/amis.tf

```diff
@@ -49,6 +49,23 @@ data "aws_ami" "rhel_8" {
   owners = ["309956199498"]
 }
 
+# identify ubuntu 14.04 ami
+data "aws_ami" "ubuntu_1404" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-trusty-14.04-amd64-server-*"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"]
```

Comment:
who is this? and would that work with other engineers? 🤔

Reply:
That owner is Canonical (the company) so it'll work fine for discovering the AMI.